nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -155,9 +155,9 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         try {
             session.exportTo(flowFile, response.getOutputStream());
             response.flushBuffer();
-        } catch (final IOException ioe) {
+        } catch (final Exception e) {
```

Comment:
You removed "IOException" from being used but didn't remove the import causing it to fail contrib check.